### ISSUE TITLE
Graphics glitch on iOS version

### ISSUE THREAD

**Issue by KarimGeiger** [matomo-org/matomo-mobile-2#5435]
On the latest iOS version, when opening the Visits Log, after going back a few pages by pressing the „Previous“ button, entries get displayed on top of each other, making them impossible to read.

![692E3B0F-5E4F-4C85-AED5-99F2EACE2866](https://user-images.githubusercontent.com/2329930/156047727-85b5b4dc-025d-4cfd-b206-2a51e0311618.png)

**Comment by tsteur** [comment]
Thanks for reporting this @KarimGeiger . Does this happen every time you open it?

**Comment by KarimGeiger** [comment]
@tsteur yes, it's reliably reproducible. Sometimes it happens on the second page, sometimes on the third or fourth. The first page is always fine.

**Comment by tsteur** [comment]
Thanks for this @KarimGeiger It took me a while to reproduce it but could see it happening once after moving a lot between pages. It happened for one row for me, and funnily it corrected itself after few seconds. Still struggling to reproduce it more often. Could you let us maybe know your iOS version and your model?

**Comment by KarimGeiger** [comment]
Sure. I’m on 15.3.1 on an iPhone 12 Pro.
A wild guess, but maybe it’s to do with how fast the server responds?

**Comment by tsteur** [comment]
Be great if you could check if this happens maybe more when being on a slower mobile data or more on a faster wifi. I couldn't reproduce any difference there so far but that doesn't mean anything. If you notice it sometimes happens more often and sometimes less often that be great to let us know @KarimGeiger it would help us better reproduce the issue and understand why this happens.

**Comment by tsteur** [comment]
Thanks for this @KarimGeiger . We'll try to reproduce it again and see if we can do something.

**Comment by tsteur** [comment]
@KarimGeiger in case you're still using the app, is this still happening? I'm still not able to reproduce it

**Comment by KarimGeiger** [comment]
@tsteur Haven't checked in a white, but after just trying it again, it's no longer happening anymore. Closing the issue.

**Comment by tsteur** [comment]
Thanks for confirming, very appreciated.
